buildpipeline/DotNet-CoreRT-Linux.json

```diff
@@ -503,6 +539,60 @@
     {
       "enabled": true,
       "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Download nuget",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "curl",
+        "arguments": "-L -k -o $(SourceFolder)/nuget.zip https://dotnet.myget.org/F/dotnet-buildtools/api/v2/package/NuGet.CommandLine/3.5.0-rc-1256",
+        "workingFolder": "",
+        "failOnStandardError": "false"
+      }
+    },
+    {
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Unzip nuget",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "unzip",
+        "arguments": "$(SourceFolder)/nuget.zip -d $(SourceFolder)/nuget",
+        "workingFolder": "",
+        "failOnStandardError": "false"
+      }
+    },
+    {
+      "enabled": false,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Publish packages",
```

Comment:
I think we should do this but not right now, I think we just want to get it off the ground first so we can test in CoreFx.  I logged https://github.com/dotnet/corert/issues/2389 to fix this.